packages/frontend/src/components/staking/components/BalanceBox.js

```diff
@@ -124,6 +124,7 @@ export default function BalanceBox({
                     <div className='title'>
                         {title && <Translate id={title}/>}
                         {info && <Tooltip translate={info}/>}
+                        {loading && <span className="animated-dots" style={{width: 16}}/>}
```

Comment:
Just curious why we need to set the `width`? It doesn't seem to make any noticeable difference when removing it.